tests/pull.bats

```diff
@@ -184,3 +184,16 @@ load helpers
   [ "$status" -eq 0 ]
   [[ "$output" =~ "ostree-repo:latest" ]]
 }
+
+
+@test "pull-all-tags" {
+  run buildah pull --all-tags alpine
```

Comment:
Adding a `--signature-policy ${TESTSDIR}/policy.json` here should fix the CI errors.

Reply:
ugh, one day I'll remember to add them after I finish testing in my sandbox.  Thanks @nalind.